src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -95,6 +95,8 @@ public function save(Key $key)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since Symfony 4.4 and will be removed in 5.0.
```

Comment:
should be `@deprecated since Symfony 4.4`, no need for the `and will` part. Same for all deprecations, see https://github.com/symfony/symfony/pull/26225

Reply:
done